### PR TITLE
Clarify password file group support

### PR DESCRIPTION
**Pull request by liggitt**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2750)

### CODE REVIEW

File: docs/admin/authentication.md [hunk 1]
```diff
@@ -115,9 +115,9 @@ and the password cannot be changed without restarting API server. Note that basi
 authentication is currently supported for convenience while we finish making the
 more secure modes described above easier to use.
 
-The basic auth file is a csv file with a minimum of 3 columns: password,
-user name, user id, followed by optional group names. Note, if you have more than
-one group the column must be double quoted e.g.
+The basic auth file is a csv file with a minimum of 3 columns: password, user name, user id.
+In 1.6+, an optional fourth column containing comma-separated group names can be specified.
```
Comment (devin-donnelly) [inline docs/admin/authentication.md]: "In Kubernetes version 1.6 and later, you can specify an optional fourth column containing comma-separated group names. If you have more than one group, you must enclose the fourth column value in double quotes ("). See the following example:"

**Comment by liggitt** [comment]
updated

**Comment by liggitt** [comment]
@deads2k PTAL for tech review

**Comment by deads2k** [comment]
lgtm

**Pull request merged**